assets/sass/patterns/organisms/content-aside.scss

```diff
@@ -29,15 +29,14 @@
   }
 
   .contextual-data__list {
```

Comment:
Can we not have this CSS in the contextual data css file rather than having it in two places

Reply:
I'm not sure I understand correctly - do we need to move just the "contextual-data__list" css to the existing file "contextual-data.scss" ?

Reply:
Done, contextual-data css is removed from the content-aside.scss file to the contextual-data.scss file